GitHubDashboard/Controllers/QueryController.cs

```diff
@@ -132,6 +132,39 @@ private async Task<IReadOnlyList<Issue>> GetIssuesAsync(string owner, string rep
             // Catch it in the calling function...
             var issues = await _gitHubClient.Issue.GetAllForRepository(owner, repository, issueRequest);
             issues = issues.Where(i => i.PullRequest == null).ToList();
+
+            // We now need to exclude the milestone
+            if (!String.IsNullOrEmpty(excludedMilestone) && !(excludedMilestone == "undefined"))
+            {
+                issues = issues.Where(i => i.Milestone == null || i.Milestone.Title != excludedMilestone).ToList();
```

Comment:
We're now realizing the list multiple times.  Maybe we should remove the `ToList` from here and line 134, and instead make it an else of the below.

Reply:
Fixed

---
In reply to: [247579075](https://github.com/Pilchie/GitHubDashboard/pull/13#discussion_r247579075) [](ancestors = 247579075)